stetho-urlconnection/src/main/java/com/facebook/stetho/urlconnection/StethoURLConnectionManager.java

```diff
@@ -37,24 +31,37 @@
  */
 @NotThreadSafe
 public class StethoURLConnectionManager {
-  private static final AtomicInteger sSequenceNumberGenerator = new AtomicInteger(0);
+  private static boolean sIsStethoPresent = isStethoPresent();
+
+  private static boolean isStethoPresent() {
```

Comment:
I like this pattern because it plays well with `static final` assignment (which I forgot to use above, oops).  Otherwise you have to use a temporary variable.

That said, I think @rickbrew 's `ReflectionUtil.tryGetClassForName` can bail us out here.